prime-angular-frontend/src/app/modules/site-registration/pages/business-licence/business-licence.component.html

```diff
@@ -46,7 +46,7 @@
 
     <section class="mb-3">
       <app-page-subheader2>
-        <ng-container appPageSubheader2Title>Doing Business As</ng-container>
+        <ng-container appPageSubheader2Title>Site Name (as it appears on your business licence)</ng-container>
```

Comment:
We may want to adjust wording since duplicated explanation to the summary below.